static/js/components/tiles/sv_ranking_units.tsx

```diff
@@ -93,11 +92,11 @@ export function SvRankingUnits(props: SvRankingUnitsProps): JSX.Element {
       chartWidth,
       chartHeight,
       chartHtml,
-      points,
-      props.sources || Array.from(rankingGroup.sources),
-      rankingGroup.svName
+      chartTitle,
+      props.sources || Array.from(rankingGroup.sources)
     );
   }
+  const chartTitle = getDownloadFilename(title, rankingGroup);
```

Comment:
Reads a little weird to have the chartTitle as a filename. It looks like tihs is the only place getDownloadFilename gets called, and it doesn't return a file extension. Perhaps rename to getChartTitle?

Reply:
Done